src/kernel/chainparams.cpp

```diff
@@ -223,8 +223,8 @@ class CTestNetParams : public CChainParams {
         consensus.vDeployments[Consensus::DEPLOYMENT_TAPROOT].nTimeout = 1628640000; // August 11th, 2021
         consensus.vDeployments[Consensus::DEPLOYMENT_TAPROOT].min_activation_height = 0; // No activation delay
 
-        consensus.nMinimumChainWork = uint256S("0x00000000000000000000000000000000000000000000076f6e7cbd0beade5d20");
-        consensus.defaultAssumeValid = uint256S("0x0000000000000004877fa2d36316398528de4f347df2f8a96f76613a298ce060"); // 2344474
+        consensus.nMinimumChainWork = uint256S("0x000000000000000000000000000000000000000000000977edb0244170858d07");
+        consensus.defaultAssumeValid = uint256S("0x0000000000000021bc50a89cde4870d4a81ffe0153b3c8de77b435a2fd3f6761"); // 2429000
```

Comment:
2429000 on testnet might be too recent according to the docs: "Testnet should be set with a height some tens of thousands back from the tip, due to reorgs there.". It's currently 428 blocks old. There are still frequent [reorgs on testnet](https://fork.observer/?network=2), with the youngest being at 2428585. However, most reorgs on testnet  I've seen are one-block reorgs and wouldn't affect this.

Reply:
It will also be much deeper once 25.x is actually released.